lib/p2p/Pool.ts

```diff
@@ -456,6 +475,14 @@ class Pool extends EventEmitter {
     });
   }
 
+  private clearOutgoingConnection = (nodePubKey: any) => {
```

Comment:
`nodePubKey` should be a `string` rather than `any`.

Edit: Although actually, if we go with a `Set` we can just use the `delete()` function it provides and wouldn't need this one.